cmd/cat/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	var lines int
+	flag.IntVar(&lines, "n", -1, "custom number of lines to print")
```

Comment:
why the default is -1?

Reply:
> why the default is -1?

because I add 1 to it when calling function